lib/extensions/postgres_cdc_rls/cdc_rls.ex

```diff
@@ -85,7 +86,8 @@ defmodule Extensions.PostgresCdcRls do
         error
 
       error ->
-        Logger.error("Error starting Postgres Extension: #{inspect(error, pretty: true)}")
+        Logger.error(%{error_code: "ErrorStartingPostgresCDC", error_message: to_log(error)})
```

Comment:
I'm not even sure what these log lines will look like? Just the map? 

Need to think about these in a logging UI like Logflare and Studio.  The entire event_message should be something kind of human readable. 

Most devs don't know what Elixir map notation is or means.

I would do something maybe like:

```elixir
Logger.error( "ErrorStartingPostgresCDC: #{to_log(error)}", error_code: "ErrorStartingPostgresCDC")
```

So the stream would be prefixed with these. Easy to see which you're getting a lot of.

And then if the error code is in the metadata we can group by it if needed.

Reply:
To be honest in console it appears fine but that is a different backend from the prod one so that is a good call.

I will change to your suggestion.